### PR TITLE
chore(offchain): bump debian image to bookworm-20230814

### PR DESCRIPTION
**Pull request by endersonmaia**

### CODE REVIEW

File: offchain/Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ COPY . /usr/src/app
 RUN cargo build --release
 
 # define runtime image
-FROM debian:bookworm-20230725-slim as runtime
+FROM debian:bookworm-20230814-slim as runtime
```
Comment (gligneul) [inline offchain/Dockerfile:37]: Could we use the latest version instead of updating it regularly?
```suggestion
FROM debian:bookworm-slim as runtime
```

Comment (endersonmaia) [reply offchain/Dockerfile:37]: We could, but it would make our builds less predictable.

Comment (gligneul) [reply offchain/Dockerfile:37]: Debian is reasonably stable; it will not impact the predictability much.

**Review by gligneul — APPROVED** [review]

File: offchain/Dockerfile (hunk 1 above)
Comment (gligneul) [reply offchain/Dockerfile:37]: I will approve this now; we can discuss this again for the next release.

**Comment by tuler** [comment]
I agree it's boring to keep track of this.
Maybe leave like this but update only when we do a minor release.

**Pull request merged**